bluebottle/clients/management/commands/bulk_import.py

```diff
@@ -252,10 +270,18 @@ def _handle_orders(self, data):
         if data['donations']:
             for don in data['donations']:
                 try:
-                    project = Project.objects.get(title=don['project'])
+                    project = Project.objects.get(slug=don['project'])
                 except Project.DoesNotExist:
-                    project = Project.objects.all()[0]
+                    print "Could not find project {0}".format(don['project'])
```

Comment:
useless nitpicking really, but it would be nice to use either log statements or use print consistently ala line 53